BurnerB/FreeMentors-V2
# [#167961996] Failing signup tests

#### What does this PR do?
Failing tests for user signup
#### Description of Task to be completed?
* Write failing tests for user signup in accordance with TDD
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ch-failing-signup-tests-167961996 
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should fail

#### What are the relevant pivotal tracker stories?
[#167961996](https://www.pivotaltracker.com/story/show/167961996)

#### Screenshots (if appropriate)
![failing signup](https://user-images.githubusercontent.com/45785786/63278237-99635500-c2af-11e9-80d8-950a10ccb300.png)
